country_data.rb

```diff
@@ -0,0 +1,67 @@
+
+module EligibilityChecker
+  def self.imf_loan_eligible?(country)
+    country.development_status == 'developing' && country.gdp < 1_000_000_000_000
```

Comment:
Let's create a module GlobalConstants, and move all the static variables to that.
and include that module in all places.